packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -1,16 +1,17 @@
 <ovh-manager-incident-banner service-name="$ctrl.serviceName">
 </ovh-manager-incident-banner>
-<oui-message data-ng-if="$ctrl.canScheduleMigration" data-type="warning">
+<oui-message data-ng-if="$ctrl.canMigrateVps2016" data-type="warning">
```

Comment:
can you make this generic so that it can be used with minimal changes in future? like remove reference to 2016. May be you can name it "canMigrateVps"